src/swapd/wallet.rs

```diff
@@ -752,8 +752,9 @@ impl Wallet {
                     }
                 } else {
                     error!(
-                        "{} | Wallet not found or not on correct state",
+                        "{} | Not correct wallet, should be Bob: {:?}",
```

Comment:
Same thing with the default wallet formatter.

Reply:
b1469847

Reply:
adc8e0ee

Reply:
ddb110cc